datalad_gooey/tests/test_fsbrowser_item.py

```diff
@@ -1,7 +1,160 @@
-
 from pathlib import Path
+
 from ..fsbrowser_item import FSBrowserItem
 
+from datalad.distribution.dataset import Dataset
+from datalad.tests.utils_pytest import (
+    assert_equal,
+    assert_in,
+    assert_not_in,
+    assert_raises,
+    with_tempfile,
+
+)
+from PySide6.QtCore import (
+    Qt,
+)
+
+from ..status_light import GooeyStatusLight
+
+
+@with_tempfile
+def test_GooeyStatusLight(path=None):
+    ds = Dataset(path).create()
+    GooeyStatusLight.__call__(dataset=ds, path=ds.pathobj)
+
 
 def test_FSBrowserItem():
-    FSBrowserItem(Path.cwd())
+    fsb_item = FSBrowserItem(Path.cwd())
+    assert_equal(fsb_item.data(0, FSBrowserItem.PathObjRole), Path.cwd())
```

Comment:
```suggestion
    assert_equal(fsb_item.pathobj, Path.cwd())
```

The property is used in pretty much all places (rather than a direct call to `data()`). It would make sense to test that instead.